context/app/static/js/pages/Diversity/Diversity.jsx

```diff
@@ -237,38 +117,26 @@ function Diversity() {
                   <TableRow>
                     <HeaderCell> </HeaderCell>
                     {headers.map((type) => (
-                      <HeaderCell> {type} </HeaderCell>
+                      <HeaderCell key={type}>{type}</HeaderCell>
                     ))}
                   </TableRow>
                 </TableHead>
                 <TableBody>
-                  <TableRow>
-                    <HeaderCell> White </HeaderCell>
-                    {age.map((type) => (
-                      <TableCell> {getCount(buckets, type, 'White')} </TableCell>
-                    ))}
-                  </TableRow>
-
-                  <TableRow>
-                    <HeaderCell> Black or African American </HeaderCell>
-                    {age.map((type) => (
-                      <TableCell> {getCount(buckets, type, 'Black or African American')} </TableCell>
-                    ))}
-                  </TableRow>
-
-                  <TableRow>
-                    <HeaderCell> Hispanic </HeaderCell>
-                    {age.map((type) => (
-                      <TableCell> {getCount(buckets, type, 'Hispanic')} </TableCell>
-                    ))}
-                  </TableRow>
+                  {['White', 'Black or African American', 'Hispanic'].map((value) => (
+                    <TableRow>
+                      <HeaderCell>{value}</HeaderCell>
+                      {age.map((type) => (
+                        <TableCell key={type}>{getCount(buckets, type, value)}</TableCell>
+                      ))}
+                    </TableRow>
+                  ))}
                 </TableBody>
               </Table>
             </StyledTableContainer>
           </ChartPaper>
 
           <DonorChart
-            donorQuery={donorRaceSexQuery}
+            donorQuery={compositeQuery(termSource('race'), termSource('blood_type'))}
```

Comment:
Same as above, all of these function calls can be pulled outside the component.

Reply:
Thanks for pointing this out... I was thinking it was simpler, but I wasn't thinking about how it changes the functionality.